backend/src/test/java/com/votogether/domain/member/service/MemberServiceTest.java

```diff
@@ -238,6 +255,123 @@ void updateDetailsSameBirthYear() {
 
     }
 
+    @Nested
+    @DisplayName("내 정보 조회를 할 때")
+    class FindMemberInfo {
+
+        @Test
+        @DisplayName("정상적으로 조회가 된다.")
+        void success() {
+            // given
+            Member member = memberTestPersister.builder().save();
+            memberMetricTestPersister.builder().member(member).save();
+
+            // when
+            MemberInfoResponse memberInfoResponse = memberService.findMemberInfo(member);
+
+            // then
+            assertSoftly(softly -> {
+                softly.assertThat(memberInfoResponse.gender()).isEqualTo(Gender.MALE);
+                softly.assertThat(memberInfoResponse.birthYear()).isEqualTo(1995);
+                softly.assertThat(memberInfoResponse.postCount()).isZero();
+                softly.assertThat(memberInfoResponse.voteCount()).isZero();
+                softly.assertThat(memberInfoResponse.hasLatestAlarm()).isFalse();
+            });
+        }
+
+        @Test
+        @DisplayName("메트릭 정보가 없는 경우 예외가 발생한다.")
+        void throwsExceptionWhenNoMetrics() {
+            // given
+            Member member = memberTestPersister.builder().save();
+
+            // when, then
+            assertThatThrownBy(() -> memberService.findMemberInfo(member))
+                    .isInstanceOf(NotFoundException.class)
+                    .hasMessage("메트릭 정보가 존재하지 않습니다.");
+        }
+
+        @Test
+        @DisplayName("게시글 내역 알림이 존재한다면 회원의 최신 알림 확인 시각과 비교하여 게시글 알림이 더 최신인 경우 true를 반환한다.")
+        void returnsTrueWhenPostAlarmIsLatest() {
+            // given
+            Member member = memberTestPersister.builder().save();
+            memberMetricTestPersister.builder().member(member).save();
+            alarmTestPersister.builder().member(member).save();
+
+            // when
+            MemberInfoResponse memberInfoResponse = memberService.findMemberInfo(member);
+
+            // then
+            assertThat(memberInfoResponse.hasLatestAlarm()).isTrue();
+        }
+
+        @Test
+        @DisplayName("신고 조치 내역 알림이 존재한다면 회원의 최신 알림 확인 시각과 비교하여 게시글 알림이 더 최신인 경우 true를 반환한다.")
+        void returnsTrueWhenReportActionAlarmIsLatest() {
+            // given
+            Member member = memberTestPersister.builder().save();
+            memberMetricTestPersister.builder().member(member).save();
+            reportActionAlarmTestPersister.builder().member(member).save();
+
+            // when
+            MemberInfoResponse memberInfoResponse = memberService.findMemberInfo(member);
+
+            // then
+            assertThat(memberInfoResponse.hasLatestAlarm()).isTrue();
+        }
+
+        @Test
+        @DisplayName("신고 조치 내역 알림과 게시글 내역 알림이 모두 존재한다면 회원의 최신 알림 확인 시각과 비교하여 신고 조치 알림이 더 최신인 경우 true를 반환한다.")
```

Comment:
```suggestion
        @DisplayName("신고 조치 내역 알림과 게시글 내역 알림이 모두 존재한다면 회원의 최신 알림 확인 시각과 비교하여 알림이 더 최신인 경우 true를 반환한다.")
```

++ 신고조치알림 이나 게시글 내역 알림 둘중 하나만 알림 확인 시각 보다 최신일때도 테스트가 추가되면 좋을거같아요

Reply:
추가하겠습니다!